automations/js/src/utils/projects.mjs

```diff
@@ -194,6 +201,7 @@ class Project {
         optionId: this.fields[fieldName].options[optionName],
       }
     )
+    this.core.debug('setCustomChoiceField response:', JSON.stringify(res))
```

Comment:
This is set to DEBUG level because it is a JSON dump of the value received from a GraphQL call. In most cases, it would be too much noise but in debugging problems, it is quite invaluable.